### DISCUSSION TITLE
想问一下各位大佬，LogicFlow能够实现连线的时候规定节点的连接数量吗？

### DISCUSSION THREAD

**Discussion by NoName1995** [didi/LogicFlow#1177] [Q&A] [answered]
现在小弟有一个需求，我在使用菱形制作判断节点的时候，想让他只能向下连接两个节点，该如何实现？

**Comment by towersxu** [comment, marked as answer]
自定义节点连线规则https://docs.logic-flow.cn/docs/#/zh/guide/basic/node?id=custom-edge-connection-rules
